sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,13 +1,23 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2020-06-10)
+### New Features
+- Support to copy a custom model from one Form Recognizer resource to another
+- Added support for AAD Authentication.
+- Raise `FormRecognizerException` when a model with `ModelStatus.Invalid` is returned from the `beginTraining()` API's
+- Raise `FormRecognizerException` when an invalid analyze status is returned from the service for recognize API's
+- Add `pageNumber` property to `FormPage` and `FormTable` model
+- Update FormRecognizer API calls to return a `List` instead of `IterableStream`.
```

Comment:
will this be a breaking change? (not sure Java...)

Reply:
yes!